core/ops.rs

```diff
@@ -167,6 +169,7 @@ impl OpState {
       resource_table: Default::default(),
       get_error_class_fn: &|_| "Error",
       gotham_state: Default::default(),
+      fast_op_error: None,
```

Comment:
This is common across all sync ops? Maybe `last_fast_call_error` is more appropriate.

Reply:
Yes, it's common. I can change the name.

What's your opinion on this being here though? Would it be better to use some `core::_ops::FastOpError(Option<Err>)` in the gotham state instead? It's sort of important that this data isn't messed with from outside the op macro as well, so any barriers for easy mistakes are appreciated.

Reply:
This is fine. OpState is just a more expensive container